src/org/thoughtcrime/securesms/connect/AccountManager.java

```diff
@@ -135,6 +134,22 @@ public void switchAccountAndStartActivity(Activity activity, int destAccountId,
         }
     }
 
+    public void switchToEncrypted(Activity activity) {
```

Comment:
If we keep this approach, it would be nice to indicate in the method declaration that this method throws a RuntimeException, (maybe a more specific IllegalStateException?)